spec/support/request_helpers.rb

```diff
@@ -4,10 +4,9 @@ module RequestHelpers
   def login(user = nil, admin: false, operator: false)
     user ||= FactoryGirl.create(:user)
 
-    case
-    when admin
+    if admin
       FactoryGirl.create(:user_role, :admin, user: user)
-    when operator
+    elsif operator
```

Comment:
__[reek]__
* [ControlParameter] is controlled by argument operator [:ref](https://github.com/troessner/reek/blob/master/docs/Control-Parameter.md)

<!-- Comment by [SideCI](https://sideci.com) -->